StaticAnalyzer/views/windows.py

```diff
@@ -199,53 +222,98 @@ def _binary_analysis(tools_dir, app_dir):
     bin_path = os.path.join(app_dir, bin_an_dic['bin'])
 
     # Execute strings command
-    # Works only in Mac and Linux by default.
-    args = ["strings", bin_path]
-    bin_an_dic['strings'] = escape(subprocess.check_output(args))
-    bin_an_dic['strings'] = bin_an_dic['strings'].replace("\n", "</br>")
+    bin_an_dic['strings'] = ""
+    str_list = list(strings(bin_path))
+    str_list = set(str_list)  # Make unique # pylint: disable-msg=R0204
+    str_list = [escape(s) for s in str_list]
+    bin_an_dic['strings'] = "</br>".join(str_list)
 
     # Execute binskim analysis if vm is available
-    if settings.WINDOWS_VM_IP is not None:
-        print "[INFO] Windows VM configured."
-        name = _upload_sample(bin_path)
-        bin_an_dic = __binskim(name, bin_an_dic)
-        bin_an_dic = __binscope(name, bin_an_dic)
+    if settings.CURRENT_PLATFROM != 'Windows':
+        if settings.WINDOWS_VM_IP is not None:
+            print "[INFO] Windows VM configured."
+            name = _upload_sample(bin_path)
+            bin_an_dic = __binskim(name, bin_an_dic)
+            bin_an_dic = __binscope(name, bin_an_dic)
+        else:
+            print "[INFO] Windowsc VM not configured in settings.py. Skipping Binskim and Binscope."
+            warning = {
+                "rule_id": "VM",
+                "status": "Info",
+                "desc": "VM is not configured. Please read the readme.md in MobSF/install/windows."
+            }
+            bin_an_dic['results'].append(warning)
     else:
-        print "[INFO] Windowsc VM not configured in settings.py. Skipping Binskim and Binscope."
-        warning = {
-            "rule_id": "VM",
-            "status": "Info",
-            "desc": "VM is not configured. Please read the readme.md in MobSF/install/windows."
-        }
-        bin_an_dic['results'].append(warning)
+        # TODO(Anpassen für windows lokal)
+        print "[INFO] Running lokal analysis."
+
+        global config
+        config = configparser.ConfigParser()
+        config.read('C:\\MobSF\\Config\\config.txt')
```

Comment:
> Since I wanted to reuse setup.py from the windows-vm C:\MobSF with the corresponding configfile, settings.py is not used.

I agree that you shouldn't be using settings.py inside Windows VM. 
But for the Host Windows Configuration It should as simple to the user that, they can make it work just by doing some config in `settings.py`.(If that is possible)

It's ok to place the config anywhere in the Windows VM.
But when it comes to MobSF running to Host OS. The config should be placed under
MobSF/MobSF/config.txt
or under `<User Home Dir>/.MobSF/config.txt` based on this: 
https://github.com/ajinabraham/Mobile-Security-Framework-MobSF/blob/master/MobSF/settings.py#L33

https://github.com/ajinabraham/Mobile-Security-Framework-MobSF/blob/master/MobSF/settings.py#L65-L71

Or merged with settings.py (Highly recommend as this will be easier for User)
Let me know if I am missing something. 

FYI, `USE_HOME` setting is to support people to ship MobSF with custom OS/ Distros like Kali, Android Tamer etc. This settings allows to keep the config, db, uploaded files etc between MobSF dir or under user Home dir.



> I could also specify the options in MobSFs settings.py, but if something changes (like a download link etc.) it would have to be changed in 2 locations. Like this, all windows-related config options are in /install/windows/config.txt and it costs minimum effort to keep windows-analysis running in the vm and locally.


but at a time the user will either on VM or host Windows right?